.travis.yml

```diff
@@ -21,10 +17,9 @@ notifications:
 
 deploy:
   provider: pypi
-  username: __token__
+  username: "__token__"
   password:
-    secure: $PYPI_TOKEN
```

Comment:
This doesn't quite work. Check **https://travis-ci.org/github/Bynder/bynder-python-sdk/builds/741252181/config**         Specifically look at "secure": "$TEST_TKN". It is just treated as a string.

more so this is a travis encrypted token so unless someone has travis digest's private key we are safe here 😄